### PR TITLE
Add operator list

### PR DESCRIPTION
**Pull request by andresmgot**
### Description of the change

Operator List view obtained checking the CRD `packagemanifests.packages.operators.coreos.com`:

Screenshot from 2020-03-06 12-02-29

The tags are the operator provider and the category.

Still needs to write more tests but sending this already to get an early review :)

### Applicable issues

  - ref #1552

### CODE REVIEW

**Comment by absoludity** [comment]
Looking great! RE:

> The tags are the operator provider and the category.

How can we manage the different sizes depending on the number of tags? Maybe we'll need to watch the overflow to avoid the issue in your image where chart tile is bigger?

File: dashboard/src/components/OperatorList/OperatorList.tsx [hunk 1]
```diff
@@ -8,28 +12,63 @@ export interface IOperatorListProps {
...
+    return (
+      <CardGrid>
+        {this.props.operators.map(operator => {
+          return (
+            <InfoCard
+              key={operator.metadata.name}
+              link={`/operators/ns/${this.props.namespace}/${operator.metadata.name}`}
+              title={operator.metadata.name}
+              // TODO(andresmgot): Icons are protected by RBAC so we need to use the backend to retrieve those
+              // icon={TBD}
```
Comment (absoludity) [inline dashboard/src/components/OperatorList/OperatorList.tsx]: Won't the RBAC be sufficient when logged in as an admin (ie. for demos) until we also store these in the backend?

**Comment by andresmgot** [comment]
> How can we manage the different sizes depending on the number of tags? Maybe we'll need to watch the overflow to avoid the issue in your image where chart tile is bigger?

It seems that the current limit (150px) is not enough. We also need to be careful because setting the limit too low it would unnecessary shorten tags that fits in one line (because the sum between a short and a long tag fits in a row). In any case, I have reduced it to 120px

File: dashboard/src/components/OperatorList/OperatorList.tsx (hunk 1 above)
Comment (andresmgot) [reply dashboard/src/components/OperatorList/OperatorList.tsx]: yes, the problem is that when using `` the img request doesn't have the authorization header. What we currently do for chart icons is to skip the authorization for logos, we need to do the same for these.

Comment (absoludity) [reply dashboard/src/components/OperatorList/OperatorList.tsx]: The request that gets to the k8s API does have the authorization header if you're using the auth proxy, as cookies get sent with any request. But yes, if you're using the token authentication then it won't. Bummer - the extra complexity to support the token auth is growing.

**Pull request merged**